interface/patient_file/encounter/forms.php

```diff
@@ -19,7 +19,7 @@
 $reviewMode = false;
 if (!empty($_REQUEST['review_id'])) {
     $reviewMode = true;
-    $encounter=$_REQUEST['review_id'];
+    $encounter=intval($_REQUEST['review_id']);
```

Comment:
Is encounter id always a number. Doesn't it become a date sometime in weird cases ? (I may be wrong on this)

Reply:
In this file encounter must be integer (encounter_id).

Reply:
sanitizeNumber()

Reply:
ok I will change that